pkg/app/piped/cloudprovider/ecs/ecs.go

```diff
@@ -0,0 +1,105 @@
+// Copyright 2020 The PipeCD Authors.
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//     http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+package ecs
+
+import (
+	"context"
+	"path/filepath"
+	"sync"
+
+	"github.com/aws/aws-sdk-go-v2/service/ecs/types"
+	"go.uber.org/zap"
+	"golang.org/x/sync/singleflight"
+
+	"github.com/pipe-cd/pipe/pkg/config"
+)
+
+const (
+	defaultTaskDefinitionFilename    = "taskdef.json"
+	defaultserviceDefinitionFilename = "servicedef.json"
```

Comment:
As the decoding function, the default format should be `.yaml`.

Reply:
I missed it!